assignment2.txt

```diff
@@ -0,0 +1,16 @@
+Sean Kee
+
+1. Yes, two ore more operators can be combined in a single line of program code.
+
+2. A logical error is when there is an error in the actual code that causes the program to not run properly, or as intended. This does not necessicarily cause the program to crash, as the code is still valid. However, syntax erros are when there is a problem with the syntax, or format, of the code, which would most likely cause the program to not run at all.
```

Comment:
"However, syntax errors are when there is a problem with the syntax, or format, of the code, which would most likely cause the program to not run at all" - mostly true, syntax errors cause the program to not COMPILE. That's the key difference.